### ISSUE TITLE
Stereo camera rendering produces the same result for both sides (centered view)

### ISSUE THREAD

**Issue by lindemeier** [GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon#539]
## Issue
Left and right camera view renderings are expected but only a center view is rendered for left and right.

## Steps to Reproduce
See the attached blender file and execute the script. The results will be written to `/tmp`.
[stereo_camera.zip](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/files/9311476/stereo_camera.zip)

## System Setup (Environment)
Ubuntu 20.04
 AMD Ryzen 9 5900X, Nvidia RTX 3060 Ti, Driver 510.85.02
Blender 3.2.2 with Rpr [v3.4.0](https://github.com/GPUOpen-LibrariesAndSDKs/RadeonProRenderBlenderAddon/tree/v3.4.0)
Used GPU and CPU

## Detailed Description
The Radeon ProRender plugin does not render stereoscopic cameras. It seems that the center position of the camera is used for rendering and used as output for left and right. 
It guess the reason is that this feature is not implemented yet. I would love to use RPR in Blender for stereoscopic rendering duet to the MaterialX library, Uber Shader and high quality and performance.
If this feature is not supported, yet, I'll try to implement on my own. If you have any pointer in how to implement that would be greatly appreciated (This would be a good start https://github.com/noeol/StereoCameraBlenderAddOn since this seems to be the base of the functionality in Blender).

### Cycles
left
![Image0001_L](https://user-images.githubusercontent.com/6230962/184221944-a091b731-2d76-4b07-98e1-839c5b1dd9d9.png)

right
![Image0001_R](https://user-images.githubusercontent.com/6230962/184222004-7c72b965-04b8-42db-9b67-187ff0cf8e07.png)

### Rpr

left
![Image0001_L](https://user-images.githubusercontent.com/6230962/184222046-5be7b3d7-2489-43a6-9958-8f79f3c439e0.png)

right
![Image0001_R](https://user-images.githubusercontent.com/6230962/184222065-390b6028-9f75-448d-a78a-2e21c51e0f4a.png)

**Comment by ARozghon** [comment]
thank you for the request, the stereo camera is not supported yet.

**Comment by lindemeier** [comment]
I've implemented a solution using bpy myself that satisfies my needs.
I've created a stereo camera rig with to cameras attached to it by local transformations.
The rig can be transformed and it updates the camera transformations automatically.

For rendering I just activate each camera individually for the scene ab render once for left and right.

The advantage of this implementation compared to blenders functionality is that the intrinsic and extrinsic parameters of each camera can be adjusted independently.
